extensions/EXT/EXT_texture_mirror_clamp_to_edge.txt

```diff
@@ -0,0 +1,151 @@
+
+Dependencies
+
+    OpenGL ES 2.0 is required.
+
+    This extension is written against the OpenGL ES 3.2 Specification.
+
+Overview
+
+    EXT_texture_mirror_clamp_to_edge extends the set of texture wrap modes to
+    include an additional mode (GL_MIRROR_CLAMP_TO_EDGE_EXT) that effectively uses
+    a texture map twice as large as the original image in which the additional
+    half of the new image is a mirror image of the original image.
+
+    This new mode relaxes the need to generate images whose opposite edges
+    match by using the original image to generate a matching "mirror image".
+    This mode allows the texture to be mirrored only once in the negative
+    s, t, and r directions.
+
+New Procedure and Functions
+
+    None
+
+New Tokens
+
+    Accepted by the <param> parameter of TexParameter{if}, SamplerParameter{if}
+    and SamplerParameter{if}v, and by the <params> parameter of
+    TexParameter{if}v, TexParameterI{i ui}v and SamplerParameterI{i ui}v when
+    their <pname> parameter is TEXTURE_WRAP_S, TEXTURE_WRAP_T, or
+    TEXTURE_WRAP_R:
+
+        MIRROR_CLAMP_TO_EDGE_EXT      0x8743 (same value as OpenGL core MIRROR_CLAMP_TO_EDGE)
+
+Additions to Chapter 8 if the OpenGL ES 3.2 Specification
+(Textures and Samplers)
+
+  In section 8.10 (Texture Parameters) modify the table entries for Table 8.16
+  (Texture parameters and their values) for TEXTURE_WRAP_S, TEXTURE_WRAP_T,
+  and TEXTURE_WRAP_R and add the following to the "Legal Values" column:
+
+    Name             Type   Legal Values
+    ---------------  ----   ------------
+    TEXTURE_WRAP_S   enum   (.. as before)
+                            MIRROR_CLAMP_TO_EDGE
+    TEXTURE_WRAP_T   enum   (.. as before)
+                            MIRROR_CLAMP_TO_EDGE
+    TEXTURE_WRAP_R   enum   (.. as before)
+                            MIRROR_CLAMP_TO_EDGE
+
+  In section 8.14.2 (Coordinate Wrapping and Texel Selection) add the
+  following row to Table 8.19 (Texel location wrap mode application):
+
+    Wrap mode                Result of wrap(coord)
+    ---------                ---------------------
+    (previous entries..)
+    MIRROR_CLAMP_TO_EDGE     min(1-1/(2*size), max(1/(2*size), abs(coord)))
+
+Additions to the GLX Specification
+
+    None
+
+GLX Protocol
+
+    None
+
+Errors
+
+    The error INVALID_ENUM is generated when TexParameter* is called
+    with a target of TEXTURE_RECTANGLE and the TEXTURE_WRAP_S,
+    TEXTURE_WRAP_T, or TEXTURE_WRAP_R parameter is set to REPEAT,
+    MIRRORED_REPEAT, or MIRROR_CLAMP_TO_EDGE.
+
+Dependencies on EXT_texture3D or equivalent
+
+    If EXT_texture3D or equivalent functionality is not implemented,
+    then the references to clamping of 3D textures in this file are
+    invalid, and references to TEXTURE_WRAP_R should be ignored.
+
+Dependencies on OpenGL ES 3.0 or equivalent
+
+    If OpenGL ES 3.0 or equivalent is not supported, then ignore all
+    references to sampler objects and SamplerParameter* functions.
+
```

Comment:
Add: 
Dependencies on OpenGL ES 3.2 or equivalent

If OpenGL ES 3.2 or equivalent is not supported, then ignore all references to the TexParameterI* and SamplerParameterI* functions.